src/coreclr/ToolBox/superpmi/superpmi-shim-collector/icorjitinfo.cpp

```diff
@@ -1710,13 +1710,23 @@ void interceptor_ICJI::getCallInfo(
     param.flags                     = flags;
     param.pResult                   = pResult;
 
+#ifdef HOST_UNIX
+    // We don't seem to be able to capture the exception code in PAL exceptions when thrown
+    // from crossgen2. So assume there will be some error, then set it to zero (no error)
+    // if the `getCallInfo` call doesn't throw.
+    param.exceptionCode = 1;
+#endif // HOST_UNIX
```

Comment:
Can you please file an issue, ideally with repro instructions?

Reply:
Created issue https://github.com/dotnet/runtime/issues/49563